api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -8650,4 +8650,196 @@
 			<column name="drug_non_coded" type="varchar(255)" />
 		</addColumn>
 	</changeSet>
+
+	<changeSet id="20151006-1530" author="bahmni">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="order_set" />
+			</not>
+		</preConditions>
+		<comment>
+			Create order_set table
+		</comment>
+		<createTable tableName="order_set">
+			<column name="order_set_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="operator" type="varchar(3)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="name" type="varchar(255)">
```

Comment:
As a part of the design call, it was decided that it can have duplicates. The name may not be unique